.env

```diff
@@ -1,6 +1,8 @@
-## .env file for making variable changes to the docker compose file ## 
-##  run docker-compose config to see the templated variables ## 
+## .env file for making variable changes to the docker compose file ##
+##  run docker-compose config to see the templated variables ##
 POSTGRES_USER=usaspending
 POSTGRES_PASSWORD=usaspender
-POSTGRES_DIRECTORY=<<directory_you_want_your_postgres_database_to_live>>
-PYTHONUNBUFFERED=0
+
+# Both of these directories should go *outside* of the root repo, or they will be mounted w/Docker
+POSTGRES_CLUSTER_DIR=../docker_db
```

Comment:
native postgres isn't a cluster. Perhaps `POSTGRES_DATA_DIR`?

Reply:
https://www.postgresql.org/docs/9.5/creating-cluster.html I think this is correct.